tests/sentry/api/endpoints/test_project_metrics_extraction_rules.py

```diff
@@ -80,6 +81,68 @@ def test_create_new_extraction_rule(self):
             assert conditions[0]["value"] == "foo:bar"
             assert conditions[1]["value"] == "baz:faz"
 
+    @django_db_all
+    @with_feature("organizations:custom-metrics-extraction-rule")
+    @patch(
+        "sentry.api.endpoints.project_metrics_extraction_rules.ProjectMetricsExtractionRulesEndpoint.create_audit_entry"
```

Comment:
Why are we mocking this, and not checking if entry is creating in database in `AuditLogEntry` table?